src/txmempool.h

```diff
@@ -58,6 +59,7 @@ class CTxMemPoolEntry
     int64_t GetTime() const { return nTime; }
     unsigned int GetHeight() const { return nHeight; }
     bool WasClearAtEntry() const { return hadNoDependencies; }
+    size_t DynamicMemoryUsage() const { return nUsageSize; }
```

Comment:
Any reason we're not counting the other member variables? Seems like there are another 53 bytes per CTMemPoolEntry?